playbook/app/pb_kits/playbook/pb_multiple_users/_multiple_users.tsx

```diff
@@ -62,18 +62,23 @@ const MultipleUsers = (props: MultipleUsersProps) => {
       {usersToDisplay.map((avatarData, index) => (
         <Avatar
             {...avatarData}
-            className="pb_multiple_users_item"
+            className="pb_multiple_uscdpers_item"
```

Comment:
Looks like you were `cd`-ing somewhere accidentally.

Reply:
oh good catch lol